frontend/src/pages/Home/Home.tsx

```diff
@@ -14,7 +17,13 @@ export const Home = () => {
         onChange={e => setFilterValue(e.target.value.toLowerCase())}
         value={filterValue}
       />
-      <PokemonList filterValue={filterValue} />
+      <Link to={`/pokedex/${pageNumber === 0 ? 0 : pageNumber - 1}`}>
+        {'<-'}
+      </Link>
+      <Link to={`/pokedex/${pageNumber === 10 ? 10 : pageNumber + 1}`}>
```

Comment:
- <!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement. -->
    
    **🧹 todo:** <subject>Remove logic from UI rendering by setting variables : A constant ROUTE_POKEDEX, previousPokedexPageIndex, nextPokedexPageIndex
    
    <discussion>